static/blog/how-lazy-loading-works.json

```diff
@@ -1 +1,243 @@
untime, the components themselves have an optimal file size."],"\n",["p",null,"Aside from that, one of my favorite things to hear is how students are being taught web development with just Ionic. No framework, no build system. Just writing plain HTML and one script tag to access all of ",["a",{"href":"https://ionicframework.com/docs/components"},"Ionic's 100+ components"],", and have them lazy-load on-demand. All of this has been made easier due to Stencil’s architecture."],"\n",["p",null,"And for browsers that do not understand ES Modules and dynamic imports, Stencil also automatically generates ES5 builds with necessary polyfills that are only downloaded by browsers that require them (such as IE11)."],"\n",["h2",{"id":"focus-on-component-authoring-let-robots-do-everything-else"},"Focus On Component Authoring. Let Robots Do Everything Else."],"\n",["p",null,"Authoring, exporting, polyfilling, bundling, packaging, distributing, consuming, etc. are all complicated tasks, especially when trying to stay performant for the end-user. Tooling improves, frameworks update, bundlers change, and new APIs make old APIs obsolete. This especially holds true when your runtime framework decides to change how it works."],"\n",["p",null,"All those hundreds of components hardcoded to a specific runtime API must be rewritten yet again. The long list of “you must”, “never”, “don’t” and best practices change daily, when in the end we’re all just trying to happily build components for apps."],"\n",["p",null,"For all these reasons, this is one of the driving factors of why Stencil was created to help Ionic move quickly. Ionic core developers focus purely on creating and maintaining great looking components to be used within millions of apps. Stencil lets the core developers put their time into building components, while the robots are focused on optimizing everything else, and adjusting to new standards."],"\n",["h2",{"id":"conclusion"},"Conclusion"],"\n",["p",null,"Each framework and bundler has their own way to lazy load modules, and has its own way to be configured, which is great for innovation and new ideas, so we welcome each tool to do it their own way."],"\n",["p",null,"But with Ionic using Stencil’s lazy-loading abilities, which is actually just using native browser APIs, Ionic is able to free itself from each framework’s custom build system, tooling and changes. This helps to meet Ionic’s goal of providing a UI library for any framework, and any version of the framework, or no framework at all, all while generating components that sit atop web-standards. And while core developers can focus on component authoring and maintenance, the compiler can focus on the tasks of optimizing and bundling."],"\n",["p",null,"Hopefully this helps detail is more about how Stencil is addressing the goal of moving the complexity of easily distributing components into the build-time tooling, and let authoring and sharing components easy for developers, and fast for end-users."]]}
```

Comment:
I'm going to add something to the backlog to review these old blogs and "do something" with them (extract info and apply to the doc site). ATM, these are inaccessible on our site